docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       NODE_ENV: development
       RAILS_LOG_TO_STDOUT: 'true'
       REMOTE_USER: blalbrit@stanford.edu
-      ROLES: sdr:administrator-role
+      ROLES: sdr:administrator-role;sdr:service-manager
```

Comment:
I don't understand why this change is necessary. This role isn't one we use in https://github.com/sul-dlss/argo/blob/main/app/models/user.rb#L9

Reply:
@jcoyne they're used here: https://github.com/sul-dlss/argo/blob/main/app/forms/apo_form.rb#L7 Without adding this role, if you create an APO as admin, you can't even *see* it in the "register items" view.